### PR TITLE
Add Git <-> Kubernetes mapping for 1.10

### PR DESCRIPTION
**Pull request by calebamiles**
We're still using Golang 1.9

/area so-crazy-it-just-might-work
/area prow

### CODE REVIEW

File: scenarios/kubernetes_verify.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 BRANCH_VERSION = {
     '1.2': '1.4',
     '1.3': '1.4',
+    '1.10': '1.9',
     'master': '1.9',
```
Comment (ixdy) [inline scenarios/kubernetes_verify.py]: change this from 1.9 to 1.10

File: scenarios/kubernetes_verify.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 BRANCH_VERSION = {
     '1.2': '1.4',
     '1.3': '1.4',
+    '1.10': '1.9',
```
Comment (ixdy) [inline scenarios/kubernetes_verify.py]: we don't need this

**Comment by krzyzacy** [comment]
this will need a new Dockerfile in https://github.com/kubernetes/test-infra/tree/master/images/kubekins-test

and probably should be part of https://github.com/kubernetes/test-infra/pull/6834
cc @cjwagner

**Comment by ixdy** [comment]
@cjwagner @BenTheElder looks like we need a kubekins-test image for 1.10 still.

**Comment by BenTheElder** [comment]
> @cjwagner @BenTheElder looks like we need a kubekins-test image for 1.10 still.

On it.

**Comment by calebamiles** [comment]
let me know when you're done @BenTheElder and I'll update this PR. Updated the PR with comments, @ixdy

**Comment by BenTheElder** [comment]
PR with images pending here: https://github.com/kubernetes/test-infra/pull/6920

**Comment by calebamiles** [comment]
Updated PTAL, @BenTheElder. Thanks!

**Comment by krzyzacy** [comment]
seems this is a subset of https://github.com/kubernetes/test-infra/pull/6921

**Comment by calebamiles** [comment]
Agreed, @krzyzacy. Closing in favor of https://github.com/kubernetes/test-infra/pull/6921.

**Pull request closed without merging**